setup.cfg

```diff
@@ -1,2 +1,40 @@
-[bdist_wheel]
-universal = 1
+[metadata]
+name = python-mimeparse
+version = attr: mimeparse.__version__
+description = A module provides basic functions for parsing mime-type names and matching them against a list of media-ranges.
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+url = https://github.com/falconry/python-mimeparse
+author = DB Tsai
+author_email = dbtsai@dbtsai.com
+maintainer = Falcon team
+maintainer_email = mail@kgriffs.com
+license = MIT
+license_file = LICENSE
+classifiers =
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Programming Language :: Python :: Implementation :: CPython
+    Programming Language :: Python :: Implementation :: PyPy
+    License :: OSI Approved :: MIT License
+    Operating System :: OS Independent
+    Development Status :: 5 - Production/Stable
+    Intended Audience :: Developers
+    Topic :: Internet :: WWW/HTTP
+    Topic :: Software Development :: Libraries :: Python Modules
+keywords =
+    mime-type
+project_urls =
+    Issue Tracker=https://github.com/falconry/python-mimeparse
+
+[options]
+python_requires = >=3.6
```

Comment:
IMHO we should keep 3.5 support as long as Falcon supports 3.5. (Even if we are vendoring.)

Reply:
I guess there are no particular reason not to do that.

Reply:
Maybe if this is implemented https://github.com/falconry/python-mimeparse/issues/40